package.json

```diff
@@ -416,7 +416,13 @@
                     "staticWebApps.appArtifactSubpath": {
                         "scope": "resource",
                         "type": "string",
-                        "description": "%staticWebApps.appArtifactSubpath%"
+                        "description": "%staticWebApps.appArtifactSubpath%",
+                        "deprecationMessage": "%staticWebApps.appArtifactSubpathDeprecated%"
+                    },
+                    "staticWebApps.outputLocationSubpath": {
```

Comment:
Seems like your pattern is _not_ to include "Location" in the setting name (even though "app", "api", and "appArtifact" all append "Location" elsewhere)

Reply:
Well, the name of the property is outputLocation.  So it went from app artifact to output location.  I'm curious what the SDK will do because they use the properties appLocation, apiLocation, and appArtifactLocation.... Will it be outputLocationLocation? 🤣

Reply:
The name of the property where? According to what?

> So it went from app artifact to output location.

I thought it went from "app artifact location" to "output location"